packages/fetch-retry/package.json

```diff
@@ -2,24 +2,40 @@
   "name": "@vercel/fetch-retry",
   "version": "5.1.3",
   "license": "MIT",
-  "devDependencies": {
-    "abort-controller": "^3.0.0",
-    "jest": "^27.5.1",
-    "node-fetch": "^2.6.7"
+  "main": "index.js",
+  "files": [
+    "index.js"
+  ],
+  "scripts": {
+    "test": "jest test",
+    "lint": "cross-env TIMING=1 eslint *.js",
+    "prettier": "prettier --write \"*.js\""
```

Comment:
This command probably doesn't need to be here, as the pre-commit hook should mean this never needs to be run.

More importantly though, Prettier knows to ignore things it can't format, so you can just do `prettier -w .`.

This goes for all instances of this command :)